tests/saw/Makefile

```diff
@@ -94,7 +94,6 @@ failure-tests : bitcode
 	@${MAKE} clean-failure-logs
 	@${MAKE} failure_tests/tls_early_ccs.log
 	@${MAKE} failure_tests/tls_missing_full_handshake.log
-	@${MAKE} failure_tests/sha_bad_magic_mod.log
```

Comment:
Why is this patch getting removed?

Reply:
This is what invoked the failure_tests/sha_%.log section in the makefile, which contains the call to the test that was removed.